From a1fa6e57baeca757b8b6cb67d437872498e5bf7b Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 25 Nov 2019 16:36:07 +0800
Subject: [PATCH] rk: clang-wrapper.py ignore atags_to_fdt.c:109

stack frame size of 4416 bytes in function 'atags_to_fdt'

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I07d22e8ca0006b8797a0cfaf42d93e46fdd4ee5c
---
 scripts/clang-wrapper.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/clang-wrapper.py b/scripts/clang-wrapper.py
index a0fa09ddf246..d650806b9d99 100755
--- a/scripts/clang-wrapper.py
+++ b/scripts/clang-wrapper.py
@@ -38,6 +38,7 @@ import sys
 import subprocess
 
 allowed_warnings = set([
+    "atags_to_fdt.c:109", # arch/arm/boot/compressed/atags_to_fdt.c:109:5: warning: stack frame size of 4416 bytes in function 'atags_to_fdt' [-Wframe-larger-than=]
  ])
 
 # Capture the name of the object file, can find it.
-- 
2.35.3

